config/epio.yml

```diff
@@ -0,0 +1,25 @@
+idspace: EPIO
+base_url: /obo/epio
+
+products:
+- epso.owl: https://raw.githubusercontent.com/SCAI-BIO/EpilepsyOntology/main/release/EPIO_merged.owl
+
+
+term_browser: ontobee
+
+example_terms:
+- EPIO_0000011
+
+entries:
+
+- prefix: /releases/
+  replacement: https://raw.githubusercontent.com/SCAI-BIO/EpilepsyOntology/main/release
```

Comment:
The point of the "releases" entry is to enable versioned releases. You are just redirecting to a release folder. If you want this to work properly, you should use formal Github releases ([example](https://github.com/pato-ontology/pato/releases)), or at the very least, create _dated subfolders_ in the release directory, like:

release/2021-12-12/EPIO_merged.owl

Note if you do that, you also have to add epio.owl specifically to release folder, else the version purls wont work (i.e. http://purl.obolibrary.org/obo/epio/releases/2021-12-12/epio.owl).

Reply:
But we did not have a release folder in the "EPSO" version is this now mandatory to have release folders ?

Reply:
No, its not mandated - but the purpose of the /releases/ section in the purl config is to provide redirects for version IRIs as the one above. You can simply remove this section if you want (although I don't recommend it) - we try to encourage people to not only have version IRIs (this is actually _required_, you cant pass the dashboard without, but you are doing that), but also have the resolve (i.e. when putting the version IRI in your browser, you should find the ontology of that version). What do you think?

Reply:
Okay if it's not mandatory I would like to leave it how it is since we have to get this running as soon as possible due to upcoming papers about the EPIO. But I'll discuss this with my team in our next weekly.

Reply:
Ok, so @jamesaoverton when you see this, you can either: 

- merge this as is
- remove the releases section

I leave this up to you, as you may have a different perspective on the matter. I requested clarification from the editorial team here: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1707